cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -172,11 +177,21 @@ public void init() throws DBException {
                 WRITE_CONSISTENCY_LEVEL_PROPERTY_DEFAULT));
 
         if ((username != null) && !username.isEmpty()) {
-          cluster = Cluster.builder().withCredentials(username, password)
-              .withPort(Integer.valueOf(port)).addContactPoints(hosts).build();
+          if ((policyname != null) && policyname.equalsIgnoreCase("roundrobin")) {
```

Comment:
Let's deduplicate the code here like this. Also, let's take the load-balancing policy full class name as the property and instance it like this:

```
Cluster.Builder builder = Cluster.builder().addContactPoints(hosts).withPort(Integer.valueOf(port));
if ((username != null) && !username.isEmpty()) {
  builder.withCredentials(username, password);
}
if ((policyname != null) && !policyname.isEmpty()) {
  .withLoadBalancingPolicy((LoadBalancingPolicy)Class.forName(policyname).newInstance());
}
cluster = builder.build();
```

Reply:
Thank you for your suggestions. I've updated the PR.